### DISCUSSION TITLE
[Readibility] list references/definitions/abbreviations at the top?

### DISCUSSION THREAD

**Discussion by will-iamalpine** [Green-Software-Foundation/sci#63] [General]
This type of info is typically in the appendix of docs, and is a bit of 'noise' at the top. Can we move this boilerplate to the end, and start with 'introduction'?

**Comment by Sealjay** [comment]
Makes sense; possibly could consider including the dictionary at build time too

**Comment by atg-abhishek** [comment]
Good point @buchananwp - I think it will help people get to the meat of the spec faster and the other material is supplementary to it all.
